src/main/java/com/dev/cinema/dao/impl/TicketDaoImpl.java

```diff
@@ -0,0 +1,34 @@
+package com.dev.cinema.dao.impl;
+
+import com.dev.cinema.dao.TicketDao;
+import com.dev.cinema.exceptions.DataProcessingException;
+import com.dev.cinema.lib.Dao;
+import com.dev.cinema.model.Ticket;
+import com.dev.cinema.util.HibernateUtil;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+@Dao
+public class TicketDaoImpl implements TicketDao {
+    @Override
+    public Ticket add(Ticket ticket) {
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.persist(ticket);
```

Comment:
let's be consistent and use the same session methods everywhere unless we have a reason to use a different method